docs/working_docs/usage.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: default
+title: Usage
+nav_order: 2
+nav_exclude: true
+parent: Working
+permalink: /working/usage
+---
+{: .warning }
+These docs are a work in progress! You are welcome to browse, but you may find features/options that are not compatible with the currently released version of OSV-Scanner. Current OSV-Scanner usage docs can be found [here](../usage). 
```

Comment:
This part seems to be repeated a lot, it seems like Jekyll has a includes function, can you have a look and see if that can be used here? https://jekyllrb.com/docs/includes/

Reply:
Thanks, I'll investigate.